### ISSUE TITLE
YACN to do list

### ISSUE THREAD

**Issue by tartavull** [seung-lab/neuroglancer#120]
# Easy stuff
Implement a new data augmentation method, e.g. black out a random block
Auxiliary inputs/outputs for floodfilling
semantics as inputs
affinity map as output
Check if regularization helps
Play with error localization windowing
Support 64-bit segmentation everywhere, and remove assumptions of consecutive seg-ids
streamline visualization process (pre-mesh volumes, activate flood-filling on client side, graph server)
# Infrastructure
Reduce number of windows in error detection to speed up development
Make everything easy to use!
document everything!
Queue examples on gpu
Increase frequency of hard examples
Train from sparse/focussed annotations. Switch to precomputed data sources?
Set up a grid search for parameters
Improve diagnostics for go/no-go decisions for error correction inference
# Error Detection
Distinguish merge and split errors
More fake errors. In particular, introduce image defects and propagate errors through to segmentation errors
Multiscale error detection
Training error detection seems to take unnecessarily long. Maybe we should pre-train on reconstruction loss and then fine-tuning on error detection loss.
Replace pixel-wise equality with earth mover's distance or Hausdorff distance
# Error Correction
Accept flood-filling result only if discriminator approves
Choose flood-filling threshold to optimize discriminator approval
Generative adversarial training
Reduce coverage of error correction
Where possible, speed up CPU operations
# Regiongraph
Produce a regiongraph with weighted edges instead of binary. This will be essential for Ran's cell splitting

**Comment by tartavull** [comment]
![image](https://user-images.githubusercontent.com/4648166/26849719-3cd60798-4ad3-11e7-845d-0fb5a6efb519.png)

Tensoboard output have now be visualize with neuroboard.py

**Comment by tartavull** [comment]
train error detection to distinguish different type of errors
train the networks to actually distinguish the three cases:
obj is a superset of something in human_labels,
obj is a subset of something in human_labels
obj is empty
1) and 2) could happen simultaneously; this means a merge and split error in the same window.

**Comment by jingpengw** [comment]
@tartavull yacn do not have blending of output patches?
